src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/contactperson/ContactPersonResourceIntTest.kt

```diff
@@ -567,5 +569,65 @@ class ContactPersonResourceIntTest : IntegrationTestBase() {
           .jsonPath("contacts[3].prisoner.firstName").isEqualTo("KWAME")
       }
     }
+
+    @Nested
+    inner class ContactRestrictions {
+      private lateinit var person: Person
+      private lateinit var prisoner: Offender
+      private lateinit var contact: OffenderContactPerson
+      private lateinit var staff: Staff
+
+      @BeforeEach
+      fun setUp() {
+        nomisDataBuilder.build {
+          staff = staff(firstName = "JANE", lastName = "STAFF")
+          person = person(
+            firstName = "JOHN",
+            lastName = "BOG",
+          )
+          prisoner = offender {
+            booking {
+              contact = contact(
+                person = person,
+              ) {
```

Comment:
Confused me at first - looks like you're reassigning person until I noticed the open brackets on the line above